samples/.NET/NUnit/.runsettings

```diff
@@ -20,4 +31,12 @@
       <!--Channel>msedge</Channel-->
     </LaunchOptions>
   </Playwright>
+    <LoggerRunSettings>
+        <Loggers>
+            <!--microsoft playwright testing service logger for reporting -->
+            <Logger friendlyName="microsoft-playwright-testing" enabled="True" />
+            <!--could enable any logger additionally -->
+            <Logger friendlyName="trx" enabled="false" />
```

Comment:
Do we need this in the samples?

Reply:
I could comment it, my motivation to have this to make it clear that users can have multiple loggers so they dont need to worry about impact to thier existing reporting system like ADO, etc